etc/config-template.toml

```diff
@@ -42,6 +42,17 @@
 ## to release memory.
 # memory-usage-limit = "0B"
 
+[quota]
+## Quota is use to add some limitation for the read write flow and then
+## gain predictable stable performance.
+## CPU quota for these front requests can use, default value is 0, it means unlimited.
+## The unit is millicpu which 1000 means 1vCPU and this config is approximate.
+# forefront-cpu-time = 0
+## Write bandwidth limitation for this TiKV instance, default value is 0 which means unlimited.
+# write-bandwidth = "0B"
+## Read bandwidth limitation for this TiKV instance, default value is 0 which means unlimited
+# read-bandwidth = "0B"
```

Comment:
I don't often see the word "forefront" in a technical context. How about all use the "foreground" prefix, or drop the prefix.

Reply:
fixed